sdk/core/azure-core-amqp/src/impl/uamqp/amqp/private/claims_based_security_impl.hpp

```diff
@@ -39,6 +39,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
         CbsTokenType type,
         std::string const& audience,
         std::string const& token,
+        Azure::DateTime const& expirationTime,
```

Comment:
since this is in the public interface should it be added as last element in the param list and have a default value ? just asking about the idea not saying to do it.

Reply:
It's an internal interface, and @antkmsft made the suggestion that I not default this *because* it's an internal interface - that forces all the callers to provide the Context whenever possible.